### PR TITLE
ADD DEGO space

### PR DESCRIPTION
**Pull request by dego-team**

### CODE REVIEW

File: spaces/dego/index.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  "name": "dego.finance",
+  "defaultView": "core",
+  "address": "0xCE92FbAC689710bCa1e1c8DB265e9B06BCB85c08",
+  "token": "0xCE92FbAC689710bCa1e1c8DB265e9B06BCB85c08",
+  "core": [
+    "0x20FE4B1eD95911487499e53355BB8f14a881D735"
+  ],
+  "min": 1000,
+  "invalid": []
+}
```
Comment (e-asia) [inline spaces/dego/index.json:13]: We have updated the format in `index.json`. 
Please add 2 new parameters `chainId` and `decimals`.
You can refer to the documentation that has been completed. https://docs.snapshot.page/guides/create-a-space#3-change-your-space-metadata

File: spaces/dego/skin.scss [hunk 2]
```diff
@@ -0,0 +1,10 @@
+.name {
+  --primary-color: #384aff;
+  --bg-color: white;
+  --text-color: #586069;
+  --link-color: #111111;
+  --heading-color: #111111;
+  --border-color: #d1d5da;
+  --header-bg: white;
+  --block-bg: transparent;
+}
```
Comment (e-asia) [inline spaces/dego/skin.scss:10]: To use the default theme you will want to delete skin.scss from your space folder.

**Review by e-asia — CHANGES_REQUESTED** [review]

**Comment by e-asia** [comment]
@dego-team You have 2 active PR with #65. You may want to close one of them.

**Comment by dego-team** [comment]
I have closed #65 and commit as your request

File: spaces/dego/index.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+  "key": "dego",
+  "symbol": "DEGO",
+  "chainId": 1, // ID of the blockchain network
+  "decimals": 18, // Number of decimals in the token
```
Comment (e-asia) [inline spaces/dego/index.json:5]: JSON standard does not allow comments. Please remove them.

**Review by e-asia — CHANGES_REQUESTED** [review]

File: spaces/dego/index.json (hunk 3 above)
Comment (dego-team) [reply spaces/dego/index.json]: sorry, I'v forgot it. I have remove them

**Review by e-asia — APPROVED** [review]

**Pull request merged**